tests/fs/test_mmap.c

```diff
@@ -176,7 +179,7 @@ int main() {
     printf("/yolo/sharedoffset.txt content=%s %d\n", buffer + offset, offset);
     fclose(fd);
   }
-
+#if !defined(NODEFS)
```

Comment:
why is this part disabled?

Reply:
For `NODEFS` files are not kept in memory like for `MEMFS` and so there is no memory allocation when e.g. appending to files. So this scenario is not a case for `NODEFS`. For example the property `stream.node.usedBytes` does not exist for `NODEFS`